pkg/model/bootstrapscript.go

```diff
@@ -374,6 +395,49 @@ func (b *BootstrapScript) Run(c *fi.Context) error {
 	return nil
 }
 
+func (b *BootstrapScript) runBottleRocket(c *fi.Context) error {
+	var caTask *fitasks.Keypair
+	for _, task := range c.AllTasks() {
+		if t, ok := task.(*fitasks.Keypair); ok && fi.StringValue(t.Name) == fi.CertificateIDCA {
+			caTask = task.(*fitasks.Keypair)
+		}
+	}
+	if caTask == nil {
+		return fmt.Errorf("CA task not found")
+	}
+	caResource := caTask.Certificate()
+	ca, err := fi.ResourceAsBytes(caResource)
```

Comment:
I do have the CA task being added to `GetDependencies` (see L233 above) but the problem is that on this line we're needing to read the contents of the CA even during the dryrun process in order to print the userdata that would be created.

I'm trying to figure out why the OIDC provider and its CA dependency do not share this same problem. It might be because the OIDC provider doesn't fetch the CA thumbprints until `RenderAWS()` whereas this is fetched in `Run()`. I'm wondering if we need to break the bootstrapscript out into its own Render function and use `DefaultDeltaRunMethod` like the cloud-specific tasks?

@justinsb any thoughts here?